src/Compilers/CSharp/Test/Syntax/Parsing/StatementParsingTests.cs

```diff
@@ -2495,6 +2807,247 @@ public void TestUsingSpecialCase3()
             Assert.NotNull(us.Statement);
         }
 
+        [Fact]
+        public void TestUsingVarSpecialCase3()
+        {
+            var text = "using f ? x, y;";
```

Comment:
What is this doing? I'm unsure what exactly is happening here.

Reply:
Ah I see. It's a type `f?` there is just a space before the `?`. 

---
In reply to: [201878919](https://github.com/dotnet/roslyn/pull/28315#discussion_r201878919) [](ancestors = 201878919)